assets/js/app/DataLayer.js

```diff
@@ -98,30 +88,89 @@ LocusZoom.DataLayer.prototype.reMap = function(){
     var promise = this.parent.parent.lzd.getData(this.parent.parent.state, this.fields); //,"ld:best"
     promise.then(function(new_data){
         this.data = new_data.body;
-        this.postget();
     }.bind(this));
     return promise;
 };
 
+/****************
+  Color Functions
+  Singleton for accessing/storing functions to apply different color schemes to data sets
+*/
+
+LocusZoom.DataLayer.ColorFunctions = (function() {
```

Comment:
I know we already talked about this, but the general numeric and categorical formatting probably won't be specific to color. They are more general patterns we would work to apply to other attributes.